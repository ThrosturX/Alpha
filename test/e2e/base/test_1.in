A 1
B 1
C 1
A 2
B 2
C 2
A 3
no, I would not like to play again.
