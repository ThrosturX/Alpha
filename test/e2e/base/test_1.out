Tic Tac Toe version 1.0.0
	 by TEAM ALPHA

   A   B   C
1:   |   |  
  --- --- ---
2:   |   |  
  --- --- ---
3:   |   |  

Player X's turn:
   A   B   C
1: X |   |  
  --- --- ---
2:   |   |  
  --- --- ---
3:   |   |  

Player O's turn:
   A   B   C
1: X | O |  
  --- --- ---
2:   |   |  
  --- --- ---
3:   |   |  

Player X's turn:
   A   B   C
1: X | O | X
  --- --- ---
2:   |   |  
  --- --- ---
3:   |   |  

Player O's turn:
   A   B   C
1: X | O | X
  --- --- ---
2: O |   |  
  --- --- ---
3:   |   |  

Player X's turn:
   A   B   C
1: X | O | X
  --- --- ---
2: O | X |  
  --- --- ---
3:   |   |  

Player O's turn:
   A   B   C
1: X | O | X
  --- --- ---
2: O | X | O
  --- --- ---
3:   |   |  

Player X's turn:
   A   B   C
1: X | O | X
  --- --- ---
2: O | X | O
  --- --- ---
3: X |   |  

The score is:
	X	O	Tie
	1	0	0

Play again? Please come again.

